events/search_built.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+import "github.com/ONSdigital/go-ns/avro"
+
+// SearchBuilt contains data related to a search that has just been built.
+type SearchBuilt struct {
```

Comment:
Consider calling this `SearchIndexBuilt` to be consistent with the schema name

Reply:
👍